pkgs/by-name/bl/blint/package.nix

```diff
@@ -0,0 +1,70 @@
+{
+  lib,
+  python3Packages,
+  fetchFromGitHub,
+  versionCheckHook,
+}:
+
+python3Packages.buildPythonApplication rec {
+  pname = "blint";
+  version = "2.4.1";
+  pyproject = true;
+
+  src = fetchFromGitHub {
+    owner = "owasp-dep-scan";
+    repo = "blint";
+    tag = "v${version}";
+    hash = "sha256-mGeC7+YzQWSlT3sW2la/f21fN8V+YoFd4fwj/PBPCMI=";
+  };
+
+  build-system = [
+    python3Packages.poetry-core
+  ];
+
+  dependencies = with python3Packages; [
+    pyyaml
+    appdirs
+    apsw
+    ar
+    custom-json-diff
+    defusedxml
+    email-validator
+    lief
+    oras
+    orjson
+    packageurl-python
+    pydantic
+    rich
+    symbolic
+  ];
+
+  pythonRelaxDeps = [
+    "apsw"
+    "symbolic"
+  ];
+
+  pythonImportsCheck = [
+    "blint"
+  ];
+
```

Comment:
btw, no versionCheckHook because there isn't a version output in help and there is no `--version`